dev/ProgressRing/ProgressRing.xaml

```diff
@@ -8,108 +8,25 @@
-        <Setter Property="Maximum" Value="100" />
+        <Setter Property="IsHitTestVisible" Value="False" />
+        <Setter Property="HorizontalAlignment" Value="Center" />
+        <Setter Property="VerticalAlignment" Value="Center" />
+        <Setter Property="MinHeight" Value="20" />
+        <Setter Property="MinWidth" Value="20" />
+        <Setter Property="IsTabStop" Value="False" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:ProgressRing">
                     <Grid x:Name="LayoutRoot" Background="Transparent">
-
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualStateGroup.Transitions>
-                                    <VisualTransition From="Indeterminate" To="Determinate">
-                                        <Storyboard>
-                                            <DoubleAnimation
-                                                Storyboard.TargetName="DeterminateProgressRingRoot"
-                                                Storyboard.TargetProperty="Opacity"
-                                                To="1"
-                                                Duration="0:0:0.15" />
-                                            <DoubleAnimation
-                                                Storyboard.TargetName="LottiePlayer"
-                                                Storyboard.TargetProperty="Opacity"
-                                                To="0"
-                                                Duration="0:0:0.15" />
-                                        </Storyboard>
-                                    </VisualTransition>
-                                </VisualStateGroup.Transitions>
-                                <VisualState x:Name="Determinate" />
-                                <VisualState x:Name="Indeterminate">
-                                    <Storyboard>
-                                        <DoubleAnimation
-                                            Storyboard.TargetName="LottiePlayer"
-                                            Storyboard.TargetProperty="Opacity"
-                                            To="1"
-                                            Duration="0:0:0.25" />
-                                        <DoubleAnimation
-                                            Storyboard.TargetName="DeterminateProgressRingRoot"
-                                            Storyboard.TargetProperty="Opacity"
-                                            To="0"
-                                            Duration="0:0:0.25" />
-                                    </Storyboard>
-                                </VisualState>
+                                <VisualState x:Name="Active" />
+                                <VisualState x:Name="Inactive" />
```

Comment:
>Inactive [](start = 53, length = 8)

why do you have these states if they don't have anything in them? as a customization point?

Reply:
For compat with wuxc.